src/common/components/survey-form/survey.form.tsx

```diff
@@ -0,0 +1,54 @@
+import { Input } from 'antd';
+import Form, { FormComponentProps } from 'antd/lib/form';
+import FormItem from 'antd/lib/form/FormItem';
+import React, { Component } from 'react';
+
+export interface SurveyFormProps extends FormComponentProps {
+    name: string;
+    description: string;
+}
+
+export class SurveyForm extends Component<SurveyFormProps, any> {
+    componentDidMount() {
+        // To disabled submit button at the beginning.
+        this.props.form.validateFields();
+    }
+
+    handleSubmit = (e: any) => {
+        e.preventDefault();
+        this.props.form.validateFields((err, values) => {
+            if (!err) {
+                console.log('Received values of form: ', values);
```

Comment:
:x:

Reply:
Please elaborate. Saving said form will have to wait until we have some kind of persistence implemented.

Reply:
I was just hinting at the `console.log`, don't take it to heart :)

Reply:
Aaah, they've been commented out now, better?

Reply:
Yes :)

Reply:
Disagree - comments are for comments, not unused debug code